python/pyspark/sql/dataframe.py

```diff
@@ -473,6 +473,13 @@ def isStreaming(self) -> bool:
         """
         return self._jdf.isStreaming()
 
+    def isEmpty(self) -> bool:
+        """Returns ``True`` if this :class:`DataFrame` is empty.
```

Comment:
Could you add `Examples` section?
Then, I think we don't need additional tests for `isEmpty` in `test_dataframe.py`. The doctet is enough.

Reply:
Done